src/gebt_poc/field_data.h

```diff
@@ -0,0 +1,110 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+
+#include "mesh.h"
+
+namespace openturbine::gebt_poc {
+
+enum class Field {
+    Coordinates,
+    Velocity,
+    Acceleration,
+    AlgorithmicAcceleration,
+    CoordinatesNext,
+    AlgorithmicAccelerationNext,
+    DeltaCoordinates,
+    Weight,
```

Comment:
Putting them as field variables has a lot of advantages.  We don't need to allocate memory for them on the fly, so we don't need to worry about race conditions as much and can avoid dynamic memory allocation in our algorithms.  This can also reduce the amount of stuff that we have to compute each iteration, since we can pre-compute stuff like mass matrices, quadrature weights, etc.  Without using scratch memory (which is complexity we don't want at this stage), using element-wise field data to store quadrature data is probably the best strategy.